### ISSUE TITLE
Problem with release on conda-forge

### ISSUE THREAD

**Issue by pbranson** [zarr-developers/numcodecs#350]
There appears to have been a problem with the latest release, see https://github.com/ocean-eddy-cpt/gcm-filters/issues/156

The shuffle codec hasn't compiled correctly it seems.

**Comment by MSanKeys963** [comment]
Cc: @jakirkham

**Comment by jakirkham** [comment]
@pbranson, can someone from your team please share a reproducer? Specifically can someone please [fill out this issue template]( https://github.com/conda-forge/numcodecs-feedstock/issues/new?assignees=&labels=bug&template=1-bug-report.yml )? Unfortunately there is not enough to go on here.

FWIW pulling a random Conda package (Linux `x86_64` for Python 3.9) from conda-forge shows the `shuffle` filter included, but that could be a different package than the one used.

We also run the full test suite on the feedstock before each release; so, would think we would have caught this, but maybe we are missing something.

**Comment by jakirkham** [comment]
Ah missed the user is on PyPy. This is currently not supported in conda-forge ( https://github.com/conda-forge/numcodecs-feedstock/pull/83 ). Are they getting the package from somewhere else?

**Comment by pbranson** [comment]
I think the test environment for gcm-filters is using mamba and the conda-forge channel. @NoraLoose are you able to provide the information required in the [comment](https://github.com/zarr-developers/numcodecs/issues/350#issuecomment-1220870499) above?

Apologies for raising the issue without sufficient information!

**Comment by jakirkham** [comment]
Right. Given there are no PyPy packages for `numcodecs` on conda-forge, the big question is what packages are being referred to and how were they installed?

**Comment by pbranson** [comment]
OK I have looked through the github actions and I think I have found where the test environment is setup and the version of numcodecs that was failing:

https://github.com/ocean-eddy-cpt/gcm-filters/runs/7838966081?check_suite_focus=true#step:5:179 
This results in
https://github.com/ocean-eddy-cpt/gcm-filters/runs/7838966081?check_suite_focus=true#step:7:48
```
E   ModuleNotFoundError: No module named 'numcodecs._shuffle'
```

And the same is for Python 3.6 -> 3.9
For example this is the version installed by conda for 3.9:
```
  + numcodecs                            0.9.1  py39h295c915_0       pkgs/main/linux-64          1 MB
```
But on closer inspection, it seems that even though the tests are meant for different Python versions, they are all installing/using python 3.9! @jbusecke I think there is something wrong with the test setup!

HTH

**Comment by jbusecke** [comment]
@pbranson thanks for the ping. I believe you are right. I have submitted a fix [here](https://github.com/ocean-eddy-cpt/gcm-filters/pull/158).
Let me know if there is anything else I can help with.
